### DISCUSSION TITLE
What is most efficient way to filter a users's inventory?

### DISCUSSION THREAD

**Discussion by ForceConstant** [joalla/discogs_client#154] [Support]
Lets say I want to get a list of all the "Vinyl" a user has in their inventory. I can loop through user.inventory and check if Format == "Vinyl" but this uses the API for each one, and for most sellers they can have 1000's so that would take a long time. Is there an faster way to get this data? Like first being able filter for the format and/or genre? 

Thanks.

**Comment by JOJ0** [comment]
Sorry no good news, I think it's the only way. It takes time...

**Comment by JOJ0** [comment]
This is the docs of the API endpoint being used in p3dc: https://www.discogs.com/developers#page:marketplace,header:marketplace-inventory

In case you find something you need that is not supported in our client yet, please open a feature request issue!
